src/vaults/vaults.utils.ts

```diff
@@ -313,6 +234,92 @@ export async function loadVaultEventPerformances(
   const cutoff = (Date.now() - ONE_DAY_MS * 21) / 1000;
   const { data } = await sdk.vaults.listHarvests({ address: vaultDefinition.vaultToken, timestamp_gte: cutoff });
 
+  return estimateVaultPerformance(chain, vaultDefinition, data);
+}
+
+export function estimateDerivativeEmission(
```

Comment:
Can you leave some docs about why is it like this etc, because otherwise it looks like some magic numbers

Reply:
added